src/rpc/blockchain.cpp

```diff
@@ -436,15 +436,15 @@ UniValue getrawmempool(const UniValue& params, bool fHelp)
         );
 
     bool fVerbose = false;
-    if (params.size() > 0)
-        fVerbose = params[0].get_bool();
+    if (request.params.size() > 0)
+        fVerbose = request.params[0].get_bool();
 
     return mempoolToJSON(fVerbose);
 }
 
-UniValue getmempoolancestors(const UniValue& params, bool fHelp)
+UniValue getmempoolancestors(const JSONRequest& request)
 {
-    if (fHelp || params.size() < 1 || params.size() > 2) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
```

Comment:
No strong opinion on this. Indeed, doing `const UniValue &params = request.params;` at the beginning of each function would reduce diff, on the other hand, I don't think prefixing `request.` everywhere is bad. It even makes the code a little bit more readable. "What params? The request params".
I'm fine with keeping it like this